AGENTS.md

```
@@ -0,0 +1,225 @@
+./gradlew :mfa:binding:testDebugUnitTestCoverage --no-daemon
+./gradlew :protect:testDebugUnitTestCoverage --no-daemon
+
+# Generate API reference docs (Dokka output → build/dokka/)
+./gradlew dokkaGenerate
+```
+
+Always add tests for new functionality and ensure all existing tests pass before opening a PR. See `CONTRIBUTING.md` for the full contribution workflow.
+
+## Module Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)